frontend/src/concepts/pipelines/content/configurePipelinesServer/utils.ts

```diff
@@ -100,11 +100,16 @@ export const createDSPipelineResourceSpec = (
       awsRecord.AWS_S3_ENDPOINT?.match(/^(?:(\w+):\/\/)?(.*)/) ?? [];
 
     return {
+      dspVersion: 'v2',
+      mlpipelineUI: {
+        image: 'quay.io/opendatahub/ds-pipelines-frontend:latest', // TODO: remove this before release
+      },
```

Comment:
We should log a Task for this effort so it's associated to the Epic.

Reply:
https://issues.redhat.com/browse/RHOAIENG-2340